vagen/envs/primitive_skill/handler.py

```
@@ -0,0 +1,636 @@
+                    result_data = {"slot": slot}
+                    images_bytes = None
+
+                    if seed is not None:
+                        obs, info = env._sync_reset(seed)
+                        result_data["obs"] = obs.get("obs_str", "")
+                        result_data["info"] = info
+                        encoded_obs = _encode_obs(obs)
+                        images_bytes = encoded_obs.get("multi_modal_input")
+
+                    result_q.put((req_id, "ok", (result_data, images_bytes)))
+                except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)